pkg/handler/collector/git/git_test.go

```diff
@@ -26,8 +26,7 @@ import (
 )
```

Comment:
Just so you know, it's better practice to make this a separate PR from the github collector piece.